infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/LogReplicationConfig.java

```diff
@@ -39,16 +45,25 @@ public class LogReplicationConfig {
     // Percentage of log data per log replication message
     public static final int DATA_FRACTION_PER_MSG = 90;
 
-    // Unique identifiers for all streams to be replicated across sites
-    private Set<String> streamsToReplicate;
+    public static final UUID REGISTRY_TABLE_ID = CorfuRuntime.getStreamID(
+            getFullyQualifiedTableName(CORFU_SYSTEM_NAMESPACE, TableRegistry.REGISTRY_TABLE_NAME));
 
-    // Streaming tags on Sink/Standby (map data stream id to list of tags associated to it)
-    private Map<UUID, List<UUID>> dataStreamToTagsMap = new HashMap<>();
+    public static final UUID PROTOBUF_TABLE_ID = CorfuRuntime.getStreamID(
+            getFullyQualifiedTableName(CORFU_SYSTEM_NAMESPACE, TableRegistry.PROTOBUF_DESCRIPTOR_TABLE_NAME));
 
     // Set of streams that shouldn't be cleared on snapshot apply phase, as these
     // streams should be the result of "merging" the replicated data (from active) + local data (on standby).
     // For instance, RegistryTable (to avoid losing local opened tables on standby)
-    private Set<UUID> mergeOnlyStreams = new HashSet<>();
+    public static final Set<UUID> MERGE_ONLY_STREAMS = new HashSet<>(Arrays.asList(
```

Comment:
it's a static final field for LR usage and should belong to the config. previously we created it in ConfigManager and passed down to lots of places, which is unclear IMO

Reply:
will we ever test a case where the source and sink have different MERGE_ONLY_STREAMS ? If not then this is fine.
Otherwise making this static could make testing tricky since within a jvm both source and sink will end up sharing this datastructure